other/number_container_system.py

```diff
@@ -0,0 +1,132 @@
+"""
+A number container system that uses binary search to
+delete and insert values into arrays with O(n logn)
+write times and O(1) read times.
```

Comment:
Let' use our 88 character limit...
```suggestion
A number container system that uses binary search to delete and insert values into
arrays with O(n logn) write times and O(1) read times.
```